src/main/java/ch/njol/skript/entity/FallingBlockData.java

```diff
@@ -112,18 +111,22 @@ protected boolean match(final FallingBlock entity) {
 		return true;
 	}
 
-	@SuppressWarnings("deprecation")
 	@Override
 	@Nullable
 	public FallingBlock spawn(Location loc, @Nullable Consumer<FallingBlock> consumer) {
 		ItemType t = CollectionUtils.getRandom(types);
 		assert t != null;
-		ItemStack i = t.getRandom();
-		if (i == null || i.getType() == Material.AIR || !i.getType().isBlock()) {
-			assert false : i;
+		Material material = t.getMaterial();
+
+		if (material == Material.AIR || !material.isBlock()) {
```

Comment:
Wouldn't this cause issues with VOID_AIR or the like (see https://github.com/SkriptLang/Skript/issues/4757)

Reply:
Interesting, what about regular air (without the check, that is)? Does that throw an error, or was this check just added cause falling air is useless? If it's the latter, we should just be consistent, either don't spawn for both, or do spawn for both. If the first, we should probably expand this check because CB is likely to notice that in the future.